### ISSUE TITLE
improve resualt for model

### ISSUE THREAD

**Issue by sfarahaninia73** [qubvel/segmentation_models#252]
i have on model below code:
#####
base_model  = Unet(
             'resnet34',
             encoder_weights='imagenet',
             activation='sigmoid',
             classes=1,
             )
inp = Input(shape=(256, 256, 3))
l1 = Conv2D(3, (1, 1))(inp)
out = base_model (l1)
model = Model(inp, out, name=base_model.name)

model.compile(
              optimizer = 'adam',
              loss = losses.binary_crossentropy, 
              metrics = ([precision, recall, iou_score, f1_score, f2_score])
 )
model.summary()
#########

and  model fit is:
history=model.fit(
          x_train,
          y_train,
          batch_size = 16,
          epochs =200,
          validation_data=(x_test, y_test),
          shuffle=True,
          callbacks = callbacks
          )

but my model in not good becase when evaluate metric for test mask and predict masks resualt is:
mean_dice 0.132
mean_precision nan
mean_recall 0.087
mean_accuracy 0.537
my train image is:(133.256.256.3)
and train is:(33.256.256.3)
please help me for give better resualt my model

**Comment by qubvel** [comment]
Hi, there are a lot of factors that influence a model quality. Read some guides how to train models, increase amount of data, use augmentations, etc., etc. 
But this is not an issue, there are a lot of examples on github, please, use search, thank you.
